src/pages/groupComparison/GroupComparisonMutationsTab.tsx

```diff
@@ -101,6 +100,15 @@ export default class GroupComparisonMutationsTab extends React.Component<
                         {MUTATIONS_TOO_MANY_GROUPS_MSG}
                     </div>
                 );
+            } else if (!this.props.store.activeMutationMapperGene) {
+                return (
+                    <div
+                        className="alert alert-info"
+                        data-test="NoMutationsAvailableAlert"
+                    >
+                        {NO_MUTATIONS_AVAILABLE_MSG}
+                    </div>
+                );
```

Comment:
lets just finish with else {} instead of closing the else and then returning something